js/mage/adminhtml/variables.js

```diff
@@ -8,7 +8,7 @@
  * @category    Mage
  * @package     Mage_Adminhtml
  * @copyright   Copyright (c) 2006-2020 Magento, Inc. (https://www.magento.com)
- * @copyright   Copyright (c) 2022 The OpenMage Contributors (https://www.openmage.org)
+ * @copyright   Copyright (c) 2022-2024 The OpenMage Contributors (https://www.openmage.org)
```

Comment:
I still dont like that. A more common thing would be "see COPYRIGHT.md" file?! And only update once and not in 99999 files? :D 
</rant over>

Reply:
that can be discussed but it's outside the scope of this PR.

Reply:
@pquerner +1

At he one hand i'm okay with it, at the other it causes changes that can be avoided (for a cleaner history).